### DISCUSSION TITLE
直接复制了elementplus的源代码，但是一直报错不知道怎么解决，求救。

### DISCUSSION THREAD

**Discussion by Nemocccc** [element-plus/element-plus#15951] [Q&A]
报错信息如下：
 - - -
ERROR  Failed to compile with 3 errors                                                                                                               10:58:39

[eslint] 
C:\Users\nemo\Desktop\Ai_System\aisystem\src\components\CommonAsider.vue
  95:25  error  Parsing error: Unexpected token, expected "," (9:25)

✖ 1 problem (1 error, 0 warnings)

You may use special comments to disable some warnings.
Use // eslint-disable-next-line to ignore the next line.
Use /* eslint-disable */ to ignore all warnings in a file.
 error  in ./src/components/CommonAsider.vue?vue&type=script&lang=ts&setup=true

Module parse failed: Unexpected token (15:23)
File was processed with these loaders:
 * ./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js
You may need an additional loader to handle the result of these loaders.
|   __expose();
|
> const handleOpen = (key: string, keyPath: string[]) => {
|     console.log(key, keyPath)
|   }

 error  in ./src/components/CommonAsider.vue?vue&type=template&id=2c35a5b5&ts=true

Module parse failed: Unexpected token (14:27)
File was processed with these loaders:
 * ./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/templateLoader.js
 * ./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js
You may need an additional loader to handle the result of these loaders.
| const _hoisted_10 = /*#__PURE__*/_createElementVNode("span", null, "Navigator Four", -1 /* HOISTED */)
|
> export function render(_ctx: any,_cache: any,$props: any,$setup: any,$data: any,$options: any) {
|   const _component_el_icon = _resolveComponent("el-icon")!
|   const _component_el_menu_item = _resolveComponent("el-menu-item")!

ERROR in ./src/components/CommonAsider.vue?vue&type=script&lang=ts&setup=true (./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/components/CommonAsider.vue?vue&type=script&lang=ts&setup=true) 15:23
Module parse failed: Unexpected token (15:23)
File was processed with these loaders:
 * ./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js
You may need an additional loader to handle the result of these loaders.
|   __expose();
| 
> const handleOpen = (key: string, keyPath: string[]) => {
|     console.log(key, keyPath)
|   }
 @ ./src/components/CommonAsider.vue?vue&type=script&lang=ts&setup=true 1:0-208 1:0-208 1:209-406 1:209-406
 @ ./src/components/CommonAsider.vue 2:0-74 3:0-69 3:0-69 6:49-55
 @ ./node_modules/.pnpm/babel-loader@8.3.0_@babel+core@7.23.9_webpack@5.90.3/node_modules/babel-loader/lib/index.js??clonedRuleSet-40.use[0]!./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/views/MainRouter.vue?vue&type=script&lang=js 1:0-57 7:4-16
 @ ./src/views/MainRouter.vue?vue&type=script&lang=js 1:0-337 1:0-337 1:338-664 1:338-664
 @ ./src/views/MainRouter.vue 2:0-61 3:0-56 3:0-56 6:49-55
 @ ./src/router/index.js 6:0-43 13:13-17
 @ ./src/main.js 4:0-39 7:8-14

ERROR in ./src/components/CommonAsider.vue?vue&type=template&id=2c35a5b5&ts=true (./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/templateLoader.js??ruleSet[1].rules[3]!./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/components/CommonAsider.vue?vue&type=template&id=2c35a5b5&ts=true) 14:27
Module parse failed: Unexpected token (14:27)
File was processed with these loaders:
 * ./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/templateLoader.js
 * ./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js
You may need an additional loader to handle the result of these loaders.
| const _hoisted_10 = /*#__PURE__*/_createElementVNode("span", null, "Navigator Four", -1 /* HOISTED */)
| 
> export function render(_ctx: any,_cache: any,$props: any,$setup: any,$data: any,$options: any) {
|   const _component_el_icon = _resolveComponent("el-icon")!
|   const _component_el_menu_item = _resolveComponent("el-menu-item")!
 @ ./src/components/CommonAsider.vue?vue&type=template&id=2c35a5b5&ts=true 1:0-337 1:0-337
 @ ./src/components/CommonAsider.vue 1:0-81 6:68-74 16:80-18:3 17:29-35 16:2-18:4
 @ ./node_modules/.pnpm/babel-loader@8.3.0_@babel+core@7.23.9_webpack@5.90.3/node_modules/babel-loader/lib/index.js??clonedRuleSet-40.use[0]!./node_modules/.pnpm/vue-loader@17.4.2_vue@3.4.19_webpack@5.90.3/node_modules/vue-loader/dist/index.js??ruleSet[0].use[0]!./src/views/MainRouter.vue?vue&type=script&lang=js 1:0-57 7:4-16
 @ ./src/views/MainRouter.vue?vue&type=script&lang=js 1:0-337 1:0-337 1:338-664 1:338-664
 @ ./src/views/MainRouter.vue 2:0-61 3:0-56 3:0-56 6:49-55
 @ ./src/router/index.js 6:0-43 13:13-17
 @ ./src/main.js 4:0-39 7:8-14

ERROR in [eslint]
C:\Users\nemo\Desktop\Ai_System\aisystem\src\components\CommonAsider.vue
  95:25  error  Parsing error: Unexpected token, expected "," (9:25)

✖ 1 problem (1 error, 0 warnings)

webpack compiled with 3 errors
- - - 

代码就是element-plus -> Navigation 导航 -> Menu 菜单 -> 侧栏的完整源代码，求救，卡好久了。

**Comment by cgx-avenue** [comment]
遇到了同样的问题，参照这个解决了，希望对你有帮助 https://blog.csdn.net/weixin_68008943/article/details/131783110

**Reply by Nemocccc** [reply]
哈哈哈感谢，我之前解决了，忘记关帖子了。
